### DISCUSSION TITLE
Archived secrets missing from Secret Scanning alerts when filtering using topics

### DISCUSSION THREAD

**Discussion by ebickle** [community/community#53886] [Code Security]
### Select Topic Area

Bug

### Body

The "Secret scanning" alerts page for an organization (https://github.com/orgs/org-name/security/alerts/secret-scanning) will show alerts from archived repositories by default.

However, if the list is filtered using the `topic:` prefix the archived repositories are hidden. Many of our teams tag their repositories using topics to identify their code and search for their alerts using the prefix. We discovered that many groups were missing their 'archived' secret scanning vulnerabilities as a result. Secrets in archived repositories are just as dangerous as non-archived, and need to be treated equivalently.

**Comment by aashah** [comment]
Hey @ebickle, sorry for the super late reply.

Is this still happening? I tested this out within a test org of ours and was able to load things just fine.

My steps to reproduce:

* Went to a repo that has secrets
* Added a topic (say `foo`)
* Went to the org view for secret scanning alerts, and filtered by `topic:foo`
* Verified the alerts show up -- ✅ 
* Archived the repo
* Redo step to search by `topic:foo`
* Verified the alerts show up -- ✅

**Comment by ebickle** [comment]
Thanks @aashah! Looks like this was fixed!
